components/vaultAssetRow/vaultAssetRow.js

```diff
@@ -22,525 +21,18 @@ import {
   ENABLE_COLLATERAL_LEND_RETURNED,
   DISABLE_COLLATERAL_LEND,
   DISABLE_COLLATERAL_LEND_RETURNED,
-  CONNECT_WALLET,
   LENDING_SUPPLY_RETURNED,
-  LENDING_BORROW_RETURNED,
 } from '../../stores/constants';
 
 import classes from './vaultAssetRow.module.css';
 
-function VaultAssetDetails({ vault, account }) {
```

Comment:
Definitely not used from within the file, and not exported so good to let it go

Reply:
Yea, was in the orginal build, but got scrapped later on.